tests/test_config.py

```diff
@@ -21,6 +21,24 @@ def test_lock_timeout(self):
     def test_key_prefix_default(self):
         self.assertEqual(self.conf.key_prefix, 'redbeat:')
 
+    def test_lock_key_default(self):
```

Comment:
To check my understanding, is this the same test as in https://github.com/sibson/redbeat/pull/109/files#diff-423890b7e3c2aa4791f9e203b7f194cbR45? If so I don't think we need to remove it, being explicit here is great.

Reply:
Yes, this tests the same thing and was added to be explicit.